src/views/RecIM/components/Helpers/index.js

```diff
@@ -1,7 +1,12 @@
 import { isValid, format } from 'date-fns';
 
 const standardDate = (date, includeTime) => {
-  if (!isValid(date)) date = Date.parse(date);
+  if (!isValid(date)) date = new Date(Date.parse(date)); // try parsing if invalid
+  if (!isValid(date)) {
+    // if still invalid (perhaps null)
+    console.log('standardDate called with an invalid or null date');
+    return;
+  }
```

Comment:
```suggestion
  if(!date) return;
  if (!isValid(date)) date = Date.parse(date);
```

Reply:
I don't know what's the point of double parsing if the issue is that the date passed is null.

Reply:
ok, still looks incredibly weird. most likely we'll have to take the console.log out. but good find.